### ISSUE TITLE
meteor update from 1.5.2 to 1.6 fails on Windows at C:\tools\cli\main.js:1501:7 (springboard)

### ISSUE THREAD

**Issue by hk86** [meteor/meteor#9292]
Hi all,

I tried to update my Meteor app from version 1.5.2 to 1.6. First I installed Chocolatey and executed choco install meteor. The I tried to update my Meteor app. This is the error I received:

```
C:\Users\USER\AppData\Local\.meteor\packages\meteor-tool\1.5.2_2\mt-os.windows.x86_32\dev_bundle\lib\node_modules\meteor-promise\promise_server.js:218
      throw error;
      ^

Error: missing tool for os.windows.x86_32 in meteor-tool@1.6.0
    at Error (native)
    at springboard (C:\tools\cli\main.js:491:11)
    at C:\tools\cli\main.js:1501:7
```

Where is this directory coming from (C:\tools\cli\)?

Best regards and thank you,

HK86

**Comment by bouchepat** [comment]
Hi @hk86,

You might want to look at leosco's post [here](https://forums.meteor.com/t/upgrade-to-1-6-failing/40184).
Good luck

**Comment by hk86** [comment]
Hi @bouchepat,

The instruction from leosco worked for me.

Thanks!

**Comment by bouchepat** [comment]
Another easier and faster alternative for solving this issue is to manually update the version of meteor in the file .meteor/release in your projects. That might help others to fix this type of issue.

Replace **METEOR@1.5.2.2** by **METEOR@1.6**.

This will fetch the updated packages on launch.

**Comment by edemaine** [comment]
I'm getting the same error as @hk86 on Windows 10 64-bit. I tried @bouchepat's approach, but then I just get this error:
```
This project says that it uses Meteor 1.6.0, but you don't have that version of Meteor installed and the Meteor
update servers don't have it either. Please edit the .meteor/release file in the project and change it to a valid
Meteor release.
```
I haven't tried leosco's solution, but is this really an acceptable workaround? I have to install Meteor manually just to update, then uninstall and switch to choco? I feel like the approach described in the 1.6.0 announcement (uninstall, delete `%appdata%/.meteor`, `choco install meteor`) should suffice, but it's not. Suggest re-opening issue.

**Comment by abernix** [comment]
There are some improvements for filesystem issues which occur during `meteor update` included in Meteor 1.5.3 and even more in Meteor 1.6, which further benefits from truly supporting 64-bit rather than working in 32-bit mode through the Windows WoW64 subsystem as it was before.  We also have a Windows CI setup through AppVeyor now too, as of Meteor 1.6 (and 1.5.3).

Hopefully the situation will be vastly improved in 1.6.
